src/hardware/mixer.cpp

```diff
@@ -707,74 +694,14 @@ static void MIXER_Mix_NoSound()
 	mixer.done=0;
 }
 
-#define INDEX_SHIFT_LOCAL 14
+static std::array<MixerFrame, MIXER_QUEUE_MAX_FRAMES> queue_buffer;
 
-static void SDLCALL MIXER_CallBack(MAYBE_UNUSED void *userdata, Uint8 *stream, int len)
+static void MIXER_QueueAudio(uint16_t len)
 {
-	memset(stream, 0, len);
```

Comment:
I like how this also got knocked out in favour of zeroing the samples after they're used on an as-needed basis.